Anature/src/application/moves/PocketSand.java

```diff
@@ -0,0 +1,21 @@
+package application.moves;
+
+import application.Anature;
+import application.enums.MoveIds;
+import application.enums.Type;
+
+public class PocketSand extends Move
+{
+
+	public PocketSand(String name, int totalMovePoints, MoveIds moveId, boolean doesDamage, boolean isPhysicalAttack, double accuracyStat, Type enumType)
+	{
```

Comment:
The constructor does not need to define variable that will never be used during construction. Please delete the variables on line 10 so it looks like public PocketSand()

You are defining these with the super call below

Reply:
Makes sense. It has been changed